k8s/dependencies/opentripplanner/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ibmjava:jre-alpine
+
+ARG operator=norrbotten
+# replace with your own key from trafiklab.se
+
+ARG apikey
+
+# fetch API key from Trafiklab
```

Comment:
You're not fetching the API key here, right? But rather the GTFS-data with the API-key. Or is it more of a todo-comment?

Reply:
It's a comment for anyone that wants to use the dockerfile - they need to provide an api key from Trafiklab.